projects/ceph.yml

```
@@ -0,0 +1,5 @@
+name: Ceph
+url: https://ceph.io/
+modules:
```

```suggestion
mixins:
```

These seem to be dashboards, not kubernetes objects